ciao-controller/instance.go

```diff
@@ -359,8 +359,8 @@ func newConfig(ctl *controller, wl *types.Workload, instanceID string, tenantID
 		userData.Hostname = "cnci-" + tenantID
 	}
 
-	// handle storage resources in workload definition
-	if wl.Storage != nil {
+	// if no instance volumes specified, use those in workload
+	if len(volumes) == 0 && wl.Storage != nil {
```

Comment:
We don't have a cohesive design intent on this agreed and documented.

If wl.storage is primarily about specifying a boot image, I think it should have precedence.  If a bootable volume is user specified, I can see that overriding the workload specified one.